### PR TITLE
CLD-692 : add image tag in README

### PR DESCRIPTION
**Pull request by rwinieski**
#### Checklist: 

-  ##### Owner:

- [ ] JIRA_ID as part of branch/PR name
- [ ] Rebase the branch with upstream
- [ ] Squashed all commits into a single commit
- [ ] Added Tests
  
- ##### Reviewer:

- [ ] Reviewed Tests
- [ ] Added to Release Wiki/Jira

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Table of contents
  * [Introduction](#Introduction)
  * [Prerequisites](#Prerequisites)
- * [Supported tags](#Supported-tags)
+ * [Tag explanation](#Tag-explanation)
```
Comment (vitalykorolev) [inline README.md:4]: This links to nowhere since the heading below wasn't updated.

Comment (vitalykorolev) [reply README.md:4]: We should also link from Tag explanation section to Image Tag section.

File: README.md [hunk 2]
```diff
@@ -971,6 +971,46 @@ docker swarm leave --force
 ```
 If the process is successful, a message saying the node has left the swarm will be displayed.
 
+# Image Tag
+
+The `marklogic` image tags allow the user to pin their applications to images for a specific release, a specific minor release, a specific major release, or the latest release of MarkLogic Server
+
+## `{ML release version}-{platform}-{ML Docker release version}`
+
+This tag points to the exact version of MarkLogic Server, the base OS, and the startup scripts version. This allows an application to pin to a very specific version of the image. The image will not be updated without incrementing either the MarkLogic Sever version or the version of the startup scripts.
```
Comment (vitalykorolev) [inline README.md:980]: I think "startup scripts" is our internal understanding and may not be clear to external audience. Perhaps it should be "Docker scripts" or "supporting scripts"?

Comment (jkerr5) [reply README.md:980]: I think people familiar with Docker images in general understand what a "startup script" is but I don't feel that strongly. Prefer "supporting scripts" if you want to change it.

**Review by vitalykorolev — APPROVED** [review]

**Pull request merged**